fluentd/configs.d/openshift/filter-k8s-meta.conf

```diff
@@ -5,7 +5,6 @@
   watch "#{ENV['K8S_METADATA_WATCH'] || 'false'}"
   bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
   ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-  include_namespace_metadata true
```

Comment:
@jcantrill, why are we no longer including the namespace metadata?

Reply:
I'm going to add it back

Reply:
we include it by default... not allowing config as to me it doesnt make sense to not have it